compiler/test/grainfmt/application_indenting.expected.gr

```diff
@@ -3,22 +3,22 @@ module ApplicationIndenting
 let flagsToWasmVal = (flag, i) => {
   let riiiiiiightsInheriting = Module64.load(structPtr, 16n)
   (
-    riiiiiiightsInheriting &
-    1N << Module64.extendI32U(Module32.fromGrain(i) >> 1n)
-  ) >
-    0N
+    riiiiiiightsInheriting
+    & 1N << Module64.extendI32U(Module32.fromGrain(i) >> 1n)
+  )
+    > 0N
```

Comment:
I feel like this formatting looks a little weird how would we feel about placing the operator on the top line in cases where a parens come first?